modules/library/render/src/main/java/org/geotools/renderer/lite/gridcoverage2d/ContrastEnhancementNode.java

```diff
@@ -115,10 +118,10 @@ public InternationalString getName() {
 		
 		//load, if it is needed the GenericPiecewise tranformation
 		try{
-			new ParameterBlockJAI(GenericPiecewise.OPERATION_NAME);
+			new ParameterBlockJAI(GenericPiecewiseOpImage.OPERATION_NAME);
 			
 		}catch (final Exception e) {
-			GenericPiecewise.register(JAI.getDefaultInstance());
+			//GenericPiecewise.register(JAI.getDefaultInstance());
```

Comment:
Commented out code... wondering if the whole try block still makes sense though, it seemed to be setup just to decide whether to register the op, or not